nevergrad/functions/multiobjective/tests/test_structures.py

```diff
@@ -0,0 +1,98 @@
+from unittest import TestCase
+
+from nevergrad.functions.multiobjective.hypervolume import VectorNode, VectorLinkedList
+
+
+class TestNode(TestCase):
```

Comment:
You should completely skip the class here and deindent `test_initialize`. With pytest there is no need of the `TestCase` class. Then in the test: `assert list(node.area) == [0.0] * dim` is easier to read, and pytest will make errors explicit.